[Windows] zed is going down the wrong path with a question mark through the terminal

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When clicking on the path in the terminal in case of an error, zed goes to the wrong path with a question mark in front, and the type hint is disabled in this file

### Environment

Zed: v0.156.0 (Zed Dev e427d83b2ea5bc6ff3805f3dee55faccd9af6751)
OS: Windows 10.0.22631
Memory: 63.2 GiB
Architecture: x86_64
GPU: AMD Radeon RX 5600 XT || AMD proprietary driver || 24.5.1 (AMD proprietary shader compiler)

### If applicable, add mockups / screenshots to help explain present your vision of the feature


https://github.com/user-attachments/assets/8f21aafb-7404-4b15-bc40-9ac4da4f304f



### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
<!-- Click below this line and paste or drag-and-drop your log-->
2024-09-30T21:18:41.6713585+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 27
2024-09-30T21:18:42.5612585+03:00 [ERROR] no snapshots found for buffer 12884902426 and server 20
2024-09-30T21:18:42.5661356+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:18:42.5662031+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:18:43.9824826+03:00 [INFO] downloading language server "json-language-server"
2024-09-30T21:18:48.1015567+03:00 [INFO] using project environment for language server LanguageServerName("json-language-server")
2024-09-30T21:18:48.1087978+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "G:\\CodeProjects\\PycharmProjects\\AutoAnswer", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\json-language-server\\node_modules/vscode-langservers-extracted/bin/vscode-json-language-server", "--stdio"]
2024-09-30T21:18:52.6256389+03:00 [INFO] Summarizing updated entries took 6.8µs
2024-09-30T21:18:52.6435435+03:00 [INFO] Initializing default prettier with plugins {}
2024-09-30T21:18:52.648822+03:00 [INFO] Initializing default prettier with plugins {}
2024-09-30T21:18:52.6632486+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6633319+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6634064+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6634649+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.663532+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6635933+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6636518+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6637102+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6637716+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6638303+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.663892+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6639529+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6640114+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6640779+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.664139+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6641975+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.666318+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6663961+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6924465+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6925269+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6925954+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6926591+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6927222+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6927828+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6928406+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6928977+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6929582+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6930155+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6930732+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6931336+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.693191+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6932489+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.693309+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6933707+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6934291+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6934894+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6935515+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6936094+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6936698+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6937274+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6937855+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6938491+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6939073+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.693965+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6940304+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6940883+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.6941465+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6942071+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.6942645+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:18:52.694322+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:18:52.6943866+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:18:52.7239+03:00 [INFO] language server 38: unregistered workspace/DidChangeWatchedFiles capability with id 94766796-6417-46b3-ba83-02cea8579ad1
2024-09-30T21:18:52.7392704+03:00 [INFO] language server 5: unregistered workspace/DidChangeWatchedFiles capability with id e2a5eb50-663b-465d-b303-e758f5616c7a
2024-09-30T21:18:52.7868027+03:00 [INFO] language server 8: unregistered workspace/DidChangeWatchedFiles capability with id 66e406f4-ff95-46ba-aed0-13861267f1f7
2024-09-30T21:18:52.8140304+03:00 [INFO] language server 20: unregistered workspace/DidChangeWatchedFiles capability with id 1492cae9-2271-4137-9e65-6e15446de4e4
2024-09-30T21:18:52.8210137+03:00 [INFO] language server 23: unregistered workspace/DidChangeWatchedFiles capability with id b6a4991d-35ce-4ae8-9eed-2dd6721b964f
2024-09-30T21:18:52.8233851+03:00 [INFO] language server 25: unregistered workspace/DidChangeWatchedFiles capability with id dc0d6de2-a89b-4950-844a-3b283be6ebde
2024-09-30T21:18:52.8306395+03:00 [INFO] language server 14: unregistered workspace/DidChangeWatchedFiles capability with id 7a10c306-6fee-4de1-b1b0-cfd772f17bf6
2024-09-30T21:18:52.8309182+03:00 [INFO] language server 31: unregistered workspace/DidChangeWatchedFiles capability with id 6a6528e7-b576-4576-b794-2dbb804a7c0d
2024-09-30T21:18:52.8311684+03:00 [INFO] language server 16: unregistered workspace/DidChangeWatchedFiles capability with id 99f69384-6947-4271-afe7-6d71c146e60b
2024-09-30T21:18:52.8379335+03:00 [INFO] language server 29: unregistered workspace/DidChangeWatchedFiles capability with id 5f376027-b16c-4261-86d5-4477a4164675
2024-09-30T21:18:52.8419313+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:18:52.8441389+03:00 [INFO] language server 27: unregistered workspace/DidChangeWatchedFiles capability with id 506c9f6d-653f-4bd5-9ec9-710e6ef07938
2024-09-30T21:18:52.850638+03:00 [INFO] language server 12: unregistered workspace/DidChangeWatchedFiles capability with id 19217060-4604-4465-9944-853c5ace9746
2024-09-30T21:18:53.0121044+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:18:53.0386534+03:00 [ERROR] no snapshots found for buffer 12884902435 and server 12
2024-09-30T21:18:53.1131863+03:00 [ERROR] no snapshots found for buffer 12884902531 and server 31
2024-09-30T21:18:53.2518252+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 27
2024-09-30T21:18:53.3130147+03:00 [ERROR] no snapshots found for buffer 12884902426 and server 20
2024-09-30T21:18:53.5268823+03:00 [INFO] Language server with id 10 sent unhandled notification taplo/didChangeSchemaAssociation:
{
  "documentUri": "file:///G:/CodeProjects/PycharmProjects/AutoAnswer/pyproject.toml",
  "schemaUri": "https://json.schemastore.org/pyproject.json",
  "meta": {
    "name": "PyProject",
    "description": "Python project metadata and configuration",
    "source": "catalog",
    "catalog_url": "https://www.schemastore.org/api/json/catalog.json"
  }
}
2024-09-30T21:18:53.751628+03:00 [INFO] Language server with id 7 sent unhandled notification taplo/didChangeSchemaAssociation:
{
  "documentUri": "file:///G:/CodeProjects/PycharmProjects/multi-merchant/pyproject.toml",
  "schemaUri": "https://json.schemastore.org/pyproject.json",
  "meta": {
    "name": "PyProject",
    "description": "Python project metadata and configuration",
    "source": "catalog",
    "catalog_url": "https://www.schemastore.org/api/json/catalog.json"
  }
}
2024-09-30T21:18:54.9961104+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:18:54.9961846+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:19:01.5963122+03:00 [INFO] Summarizing updated entries took 5.7µs
2024-09-30T21:19:01.7685234+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:19:06.217443+03:00 [ERROR] Error { code: HRESULT(0x80070490), message: "Элемент не найден." }
2024-09-30T21:19:12.3360907+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:19:12.5137929+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:19:14.9844034+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:19:20.8604032+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\bot\\handlers\\common\\payment.py"]
2024-09-30T21:19:20.9500597+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\bot\\handlers\\common\\payment.py", id: 42
2024-09-30T21:19:20.9502739+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\bot\\handlers\\common\\payment.py", id: 43
2024-09-30T21:19:20.9584224+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T21:19:20.9584797+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T21:19:20.9586269+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\bot\\handlers\\common", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T21:19:20.9943032+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\bot\\handlers\\common", args: ["server"]
2024-09-30T21:19:20.9978354+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:19:21.0384257+03:00 [ERROR] no snapshots found for buffer 12884902275 and server 43
2024-09-30T21:19:21.2195732+03:00 [INFO] language server 42: unregistered workspace/DidChangeWatchedFiles capability with id 4f3a4bc7-6068-40c4-b335-cc42fd99e927
2024-09-30T21:19:21.7355262+03:00 [ERROR] no snapshots found for buffer 12884902275 and server 42
2024-09-30T21:19:23.8861782+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:19:23.8862483+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:19:41.3287041+03:00 [INFO] Summarizing updated entries took 5.5µs
2024-09-30T21:19:41.343609+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 28
2024-09-30T21:19:41.5299964+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:19:41.6697337+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 27
2024-09-30T21:19:45.3679286+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:19:45.3837959+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 28
2024-09-30T21:19:45.6359877+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:19:45.6991244+03:00 [ERROR] no snapshots found for buffer 4221952865307 and server 27
2024-09-30T21:19:46.1988358+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:19:53.9616884+03:00 [INFO] Summarizing updated entries took 8.9µs
2024-09-30T21:19:53.9821011+03:00 [INFO] Summarizing updated entries took 4.3µs
2024-09-30T21:19:57.1282616+03:00 [INFO] Summarizing updated entries took 6.4µs
2024-09-30T21:19:57.1584887+03:00 [INFO] Summarizing updated entries took 4.9µs
2024-09-30T21:20:39.8164059+03:00 [INFO] Using surface present mode FIFO_RELAXED
2024-09-30T21:20:47.1407122+03:00 [INFO] Using surface present mode FIFO_RELAXED
2024-09-30T21:21:53.7983656+03:00 [ERROR] Процесс не может получить доступ к файлу, так как этот файл занят другим процессом. (os error 32)
2024-09-30T21:23:00.8246398+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:23:00.824725+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:23:31.7207851+03:00 [INFO] Summarizing updated entries took 6.8µs
2024-09-30T21:23:31.9087764+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:24:11.1679774+03:00 [ERROR] Error { code: HRESULT(0x80070490), message: "Элемент не найден." }
2024-09-30T21:24:11.1680739+03:00 [ERROR] Error { code: HRESULT(0x80070490), message: "Элемент не найден." }
2024-09-30T21:24:11.1681566+03:00 [ERROR] Error { code: HRESULT(0x80070490), message: "Элемент не найден." }
2024-09-30T21:27:40.2366438+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Initializing", status: Complete, percent_complete: None })
2024-09-30T21:27:40.2367285+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: None })
2024-09-30T21:29:21.7271339+03:00 [INFO] Summarizing updated entries took 8.5µs
2024-09-30T21:29:21.7800101+03:00 [INFO] Summarizing updated entries took 7.7µs
2024-09-30T21:30:13.9995579+03:00 [ERROR] connection error: delay between messages too long
2024-09-30T21:30:13.9996232+03:00 [INFO] set status on client 118327: ConnectionLost
2024-09-30T21:30:14.0394626+03:00 [INFO] set status on client 118327: Reauthenticating
2024-09-30T21:30:14.0395253+03:00 [INFO] set status on client 118327: Reconnecting
2024-09-30T21:30:15.2098993+03:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-09-30T21:30:15.7849288+03:00 [INFO] add connection to peer
2024-09-30T21:30:15.7849735+03:00 [INFO] add_connection;
2024-09-30T21:30:15.7850427+03:00 [INFO] waiting for server hello
2024-09-30T21:30:15.7856756+03:00 [INFO] got server hello
2024-09-30T21:30:15.7857196+03:00 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 8 }, peer id: PeerId { owner_id: 545, id: 4469157 })
2024-09-30T21:30:15.7857869+03:00 [INFO] set status on client 118327: Connected { peer_id: PeerId { owner_id: 545, id: 4469157 }, connection_id: ConnectionId { owner_id: 0, id: 8 } }
2024-09-30T21:30:18.8651612+03:00 [ERROR] Процесс не может получить доступ к файлу, так как этот файл занят другим процессом. (os error 32)
2024-09-30T21:30:23.4958068+03:00 [INFO] Summarizing updated entries took 6.7µs
2024-09-30T21:30:23.5435454+03:00 [INFO] Summarizing updated entries took 5.7µs
2024-09-30T21:30:26.3986715+03:00 [INFO] Summarizing updated entries took 5.5µs
2024-09-30T21:30:26.4397891+03:00 [INFO] Summarizing updated entries took 5.1µs
2024-09-30T21:30:29.1851512+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:30:29.2393101+03:00 [INFO] Summarizing updated entries took 3.9µs
2024-09-30T21:30:38.1494266+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:30:42.0466697+03:00 [INFO] Summarizing updated entries took 6.1µs
2024-09-30T21:30:42.1143751+03:00 [INFO] Summarizing updated entries took 5µs
2024-09-30T21:30:42.1837558+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:30:42.2366883+03:00 [INFO] Summarizing updated entries took 5.6µs
2024-09-30T21:30:42.9148657+03:00 [INFO] Summarizing updated entries took 7.6µs
2024-09-30T21:30:42.9714099+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:30:47.5650879+03:00 [INFO] Summarizing updated entries took 8.1µs
2024-09-30T21:30:47.6086818+03:00 [INFO] Summarizing updated entries took 6.7µs
2024-09-30T21:31:01.0303447+03:00 [INFO] Summarizing updated entries took 7µs
2024-09-30T21:31:01.075583+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:33:11.745104+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:33:11.7451941+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:33:14.7039246+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:33:14.7039949+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:35:08.4288652+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:35:08.4689353+03:00 [INFO] Summarizing updated entries took 5.1µs
2024-09-30T21:35:10.1744084+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:35:14.0587784+03:00 [INFO] Summarizing updated entries took 7µs
2024-09-30T21:35:14.1633969+03:00 [INFO] Summarizing updated entries took 7.6µs
2024-09-30T21:35:14.2279427+03:00 [INFO] Summarizing updated entries took 6.8µs
2024-09-30T21:35:14.2541109+03:00 [INFO] Summarizing updated entries took 7.2µs
2024-09-30T21:35:14.3127788+03:00 [INFO] Summarizing updated entries took 5.7µs
2024-09-30T21:35:14.9348203+03:00 [INFO] Summarizing updated entries took 9.5µs
2024-09-30T21:35:14.9922028+03:00 [INFO] Summarizing updated entries took 6.9µs
2024-09-30T21:35:19.212494+03:00 [INFO] Summarizing updated entries took 7.2µs
2024-09-30T21:35:19.2612394+03:00 [INFO] Summarizing updated entries took 6.1µs
2024-09-30T21:35:19.3097524+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:35:19.3562275+03:00 [INFO] Summarizing updated entries took 6.3µs
2024-09-30T21:35:36.6037309+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:35:36.6211144+03:00 [ERROR] no snapshots found for buffer 12884902435 and server 13
2024-09-30T21:35:36.8484+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:35:36.8857946+03:00 [ERROR] no snapshots found for buffer 12884902435 and server 12
2024-09-30T21:35:41.9653571+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:35:43.7268372+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:35:43.743067+03:00 [ERROR] no snapshots found for buffer 12884902435 and server 13
2024-09-30T21:35:44.2280529+03:00 [ERROR] no snapshots found for buffer 12884902435 and server 12
2024-09-30T21:35:44.2283036+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:35:49.286781+03:00 [INFO] Summarizing updated entries took 5.9µs
2024-09-30T21:35:49.7388657+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:37:01.5985401+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:37:15.1660069+03:00 [ERROR] stream did not contain valid UTF-8
2024-09-30T21:37:19.7928968+03:00 [ERROR] Процесс не может получить доступ к файлу, так как этот файл занят другим процессом. (os error 32)
2024-09-30T21:37:22.2592445+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:37:51.3451139+03:00 [ERROR] Процесс не может получить доступ к файлу, так как этот файл занят другим процессом. (os error 32)
2024-09-30T21:37:59.368096+03:00 [INFO] Summarizing updated entries took 5.6µs
2024-09-30T21:37:59.4114239+03:00 [INFO] Summarizing updated entries took 5µs
2024-09-30T21:38:00.1683874+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:38:03.9281082+03:00 [INFO] Summarizing updated entries took 9.5µs
2024-09-30T21:38:03.9304616+03:00 [ERROR] error processing "G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\database\\db.sqlite3-journal": Не удается найти указанный файл. (os error 2)
2024-09-30T21:38:03.9754259+03:00 [INFO] Summarizing updated entries took 4.5µs
2024-09-30T21:38:04.0438541+03:00 [INFO] Summarizing updated entries took 8.3µs
2024-09-30T21:38:04.1085457+03:00 [INFO] Summarizing updated entries took 7µs
2024-09-30T21:38:04.1513687+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:38:04.8221631+03:00 [INFO] Summarizing updated entries took 4.8µs
2024-09-30T21:38:04.8812338+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:38:06.5682301+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:38:06.613838+03:00 [INFO] Summarizing updated entries took 6.7µs
2024-09-30T21:39:46.7753266+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:39:46.8011509+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:39:47.0657958+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:39:47.1017311+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:39:49.729514+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:39:49.7295964+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:39:49.7296891+03:00 [ERROR] failed to get git blame data: Failed to blame ".venv\\Lib\\site-packages\\sqlalchemy\\sql\\selectable.py"

Caused by:
    Идет закрытие канала. (os error 232)
2024-09-30T21:39:49.9141153+03:00 [INFO] Summarizing updated entries took 6.2µs
2024-09-30T21:39:49.9440748+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:39:50.1979529+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:39:50.2112072+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:40:06.94571+03:00 [INFO] Summarizing updated entries took 5.7µs
2024-09-30T21:40:07.1804884+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:40:18.3023292+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:40:18.3522017+03:00 [INFO] Summarizing updated entries took 5.2µs
2024-09-30T21:40:38.8673712+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:40:39.1534778+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:40:49.7499217+03:00 [INFO] Summarizing updated entries took 6.9µs
2024-09-30T21:40:49.9975913+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:40:50.9739718+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:40:52.1075562+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:40:52.9501413+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:41:19.7540042+03:00 [INFO] Summarizing updated entries took 5.5µs
2024-09-30T21:41:19.7716236+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:41:19.9935763+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:41:20.0353927+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:41:23.9929297+03:00 [INFO] Summarizing updated entries took 5.1µs
2024-09-30T21:41:24.0114346+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:41:25.1294378+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:41:25.1299953+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:41:26.8675333+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:41:26.8842131+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:41:27.1333056+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:41:27.1594135+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:41:37.0738894+03:00 [INFO] Summarizing updated entries took 7.5µs
2024-09-30T21:41:37.113928+03:00 [INFO] Summarizing updated entries took 3.9µs
2024-09-30T21:41:40.4776719+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:41:44.3286706+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:41:44.432211+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:41:44.4505914+03:00 [INFO] Summarizing updated entries took 4.3µs
2024-09-30T21:41:44.5106884+03:00 [INFO] Summarizing updated entries took 4.6µs
2024-09-30T21:41:44.7107031+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:41:45.1920276+03:00 [INFO] Summarizing updated entries took 7.8µs
2024-09-30T21:41:45.2348035+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:41:47.8685521+03:00 [INFO] Summarizing updated entries took 6.4µs
2024-09-30T21:41:47.9107742+03:00 [INFO] Summarizing updated entries took 4.5µs
2024-09-30T21:43:46.8135374+03:00 [INFO] Summarizing updated entries took 7.2µs
2024-09-30T21:43:46.8306415+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:43:47.1080512+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:43:47.1221815+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:43:58.108992+03:00 [INFO] Summarizing updated entries took 5.6µs
2024-09-30T21:43:58.1135547+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 24
2024-09-30T21:43:58.7676616+03:00 [ERROR] no snapshots found for buffer 12884902253 and server 23
2024-09-30T21:43:58.7678486+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:44:05.7881585+03:00 [INFO] Summarizing updated entries took 6.3µs
2024-09-30T21:44:05.8721184+03:00 [INFO] Summarizing updated entries took 3.8µs
2024-09-30T21:44:11.1631762+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:44:15.5086739+03:00 [INFO] Summarizing updated entries took 6.2µs
2024-09-30T21:44:15.7775884+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:44:15.8062733+03:00 [INFO] Summarizing updated entries took 5.5µs
2024-09-30T21:44:15.8272973+03:00 [INFO] Summarizing updated entries took 4µs
2024-09-30T21:44:15.8322616+03:00 [INFO] Summarizing updated entries took 5.5µs
2024-09-30T21:44:16.0619442+03:00 [INFO] Summarizing updated entries took 5.2µs
2024-09-30T21:44:16.1190814+03:00 [INFO] Summarizing updated entries took 4.4µs
2024-09-30T21:44:21.7903103+03:00 [INFO] Summarizing updated entries took 8.9µs
2024-09-30T21:44:21.8345402+03:00 [INFO] Summarizing updated entries took 8.4µs
2024-09-30T21:45:16.1594356+03:00 [INFO] Summarizing updated entries took 6µs
2024-09-30T21:45:16.393889+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:45:23.0180637+03:00 [ERROR] failed to get git blame data: Failed to blame ".venv\\Lib\\site-packages\\aiogram\\client\\bot.py"

Caused by:
    Идет закрытие канала. (os error 232)
2024-09-30T21:45:47.6675389+03:00 [INFO] Summarizing updated entries took 5.9µs
2024-09-30T21:45:49.4958954+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:45:49.4961214+03:00 [INFO] Summarizing updated entries took 4.4µs
2024-09-30T21:45:49.7237085+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:45:57.0837608+03:00 [INFO] Creating texture 0x2ac266c1dc0 of size 1232x1024x1 and format R16Float, name 'atlas', handle 23
2024-09-30T21:46:14.6415479+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:46:14.9066223+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:46:16.6560232+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:46:16.6560955+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:46:16.6959539+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:47:02.0977762+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:47:02.0978606+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:47:45.7592261+03:00 [INFO] Summarizing updated entries took 7.9µs
2024-09-30T21:47:46.007388+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:48:07.3003426+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:48:07.6003632+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:48:12.0427172+03:00 [INFO] Summarizing updated entries took 5.4µs
2024-09-30T21:48:12.2866984+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:48:28.6356129+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:48:30.8993527+03:00 [INFO] Summarizing updated entries took 5.3µs
2024-09-30T21:48:31.1314798+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:48:53.8584592+03:00 [INFO] Summarizing updated entries took 5.2µs
2024-09-30T21:48:54.0871606+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:48:59.2027346+03:00 [INFO] Summarizing updated entries took 6.5µs
2024-09-30T21:48:59.4596756+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:49:01.7403442+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:49:01.9869409+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:49:05.115106+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:50:45.8043154+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:51:41.1265157+03:00 [INFO] Summarizing updated entries took 7.5µs
2024-09-30T21:51:41.1456572+03:00 [ERROR] no snapshots found for buffer 12884902426 and server 21
2024-09-30T21:51:41.9037409+03:00 [ERROR] no snapshots found for buffer 12884902426 and server 20
2024-09-30T21:51:41.9042119+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:51:45.3654159+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:52:10.3432458+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:52:10.3433356+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:52:12.5938384+03:00 [INFO] Summarizing updated entries took 5.2µs
2024-09-30T21:52:13.350666+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:52:14.2827259+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:52:28.8871669+03:00 [INFO] Summarizing updated entries took 8µs
2024-09-30T21:52:28.9281309+03:00 [INFO] Summarizing updated entries took 5.8µs
2024-09-30T21:53:44.2381976+03:00 [INFO] Using Windows API for pseudoconsole
2024-09-30T21:54:53.8504508+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]
2024-09-30T21:54:53.9577889+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 44
2024-09-30T21:54:53.9579798+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 45
2024-09-30T21:54:53.9655246+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T21:54:53.9655843+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T21:54:53.9657462+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T21:54:54.0031145+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["server"]
2024-09-30T21:54:54.0192347+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:54:54.0341178+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:54:54.2192147+03:00 [INFO] language server 44: unregistered workspace/DidChangeWatchedFiles capability with id 46e552b0-970f-4912-9ea2-0ded1de5b711
2024-09-30T21:54:54.7089898+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:54:55.6485302+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T21:54:55.6486029+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T21:55:01.4097032+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:01.5285961+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:01.684541+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:01.7533232+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:02.0305004+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:02.4947585+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:02.7592194+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:03.4619047+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:03.7274801+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:13.0248197+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:13.1957421+03:00 [INFO] Summarizing updated entries took 6.3µs
2024-09-30T21:55:13.2878869+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:13.493615+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:55:43.8737534+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:44.1394384+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:44.2110153+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:44.4758107+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:48.6386898+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:48.8991495+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:49.1509359+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.174739+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.2060496+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.2504727+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.4537781+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.6111843+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.7690064+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:49.9033825+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.0179997+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.083546+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.3318009+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.3988195+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.5782658+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:50.8319143+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:51.3209069+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:51.4035321+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:51.4563286+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:51.7140541+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:51.9739889+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:52.1535765+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:52.2226012+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:52.2660567+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:52.3737661+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:52.6237149+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:53.2435593+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:53.3303046+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:53.3774681+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:53.6246336+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:54.6962577+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:54.9536832+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:55.8667083+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:55:56.1285463+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 44
2024-09-30T21:55:59.7560843+03:00 [INFO] Summarizing updated entries took 5.2µs
2024-09-30T21:56:00.0035866+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:56:04.7242966+03:00 [INFO] Summarizing updated entries took 5µs
2024-09-30T21:56:04.9780381+03:00 [ERROR] failed to fetch cached embeddings via cloud model

Caused by:
    RPC request GetCachedEmbeddings failed: permission denied
2024-09-30T21:56:07.5048834+03:00 [ERROR] oneshot canceled
2024-09-30T21:56:07.5333278+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 45
2024-09-30T21:56:08.0090745+03:00 [ERROR] no such worktree
2024-09-30T21:56:08.0091597+03:00 [ERROR] no such worktree
2024-09-30T21:56:08.009208+03:00 [ERROR] no such worktree
2024-09-30T21:56:08.0092552+03:00 [ERROR] no such worktree
2024-09-30T21:56:52.3554006+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\.venv\\Lib\\site-packages\\apscheduler\\executors\\base_py3.py"]
2024-09-30T21:56:52.4541251+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\.venv\\Lib\\site-packages\\apscheduler\\executors\\base_py3.py", id: 46
2024-09-30T21:56:52.4543518+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\.venv\\Lib\\site-packages\\apscheduler\\executors\\base_py3.py", id: 47
2024-09-30T21:56:52.461555+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T21:56:52.461614+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T21:56:52.4621531+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\.venv\\Lib\\site-packages\\apscheduler\\executors", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T21:56:52.4720197+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\.venv\\Lib\\site-packages\\apscheduler\\executors", args: ["server"]
2024-09-30T21:56:52.5034271+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T21:56:52.5035041+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T21:56:52.7066731+03:00 [INFO] language server 46: unregistered workspace/DidChangeWatchedFiles capability with id 5e16639a-6c7d-42b1-889c-66e35926209a
2024-09-30T21:57:14.2165646+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]
2024-09-30T21:57:14.3277498+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 48
2024-09-30T21:57:14.32796+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 49
2024-09-30T21:57:14.3351133+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T21:57:14.3351708+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T21:57:14.3352924+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T21:57:14.3474385+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["server"]
2024-09-30T21:57:14.3800183+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T21:57:14.3981301+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 49
2024-09-30T21:57:14.5800137+03:00 [INFO] language server 48: unregistered workspace/DidChangeWatchedFiles capability with id cbe14da9-1d59-437d-ba71-06955d7f0887
2024-09-30T21:57:15.0436026+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 48
2024-09-30T21:58:36.0828557+03:00 [INFO] open paths []
2024-09-30T21:58:39.1568409+03:00 [ERROR] oneshot canceled
2024-09-30T21:58:39.1863601+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 49
2024-09-30T21:58:44.5884023+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]
2024-09-30T21:58:44.6904396+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 50
2024-09-30T21:58:44.6906261+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 51
2024-09-30T21:58:44.6978962+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T21:58:44.6979527+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T21:58:44.6984113+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T21:58:44.7323761+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["server"]
2024-09-30T21:58:44.7734807+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 51
2024-09-30T21:58:44.944131+03:00 [INFO] language server 50: unregistered workspace/DidChangeWatchedFiles capability with id 657d651f-66bf-4a1a-bb51-519b773a095e
2024-09-30T21:58:45.4295046+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 50
2024-09-30T22:01:15.9664393+03:00 [ERROR] oneshot canceled
2024-09-30T22:01:15.9919626+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 51
2024-09-30T22:01:27.1894533+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]
2024-09-30T22:01:27.318666+03:00 [INFO] attempting to start language server "pyright", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 52
2024-09-30T22:01:27.3188694+03:00 [INFO] attempting to start language server "ruff", path: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py", id: 53
2024-09-30T22:01:27.3266643+03:00 [ERROR] Неверно задано имя папки. (os error 267)
2024-09-30T22:01:27.3267284+03:00 [INFO] using project environment for language server LanguageServerName("pyright")
2024-09-30T22:01:27.3268638+03:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["C:\\Users\\taima\\AppData\\Local\\Zed\\languages\\pyright\\node_modules/pyright/langserver.index.js", "--stdio"]
2024-09-30T22:01:27.3649509+03:00 [INFO] starting language server process. binary path: "C:\\Users\\taima\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\ruff.exe", working directory: "\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs", args: ["server"]
2024-09-30T22:01:27.3993058+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 53
2024-09-30T22:01:27.5836479+03:00 [INFO] language server 52: unregistered workspace/DidChangeWatchedFiles capability with id a281087b-3c11-4d10-a30c-a4e7696a6e86
2024-09-30T22:01:28.0725636+03:00 [ERROR] no snapshots found for buffer 12884902617 and server 52
2024-09-30T22:01:34.4783327+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disable_reason", value: String(".gitignore") })
2024-09-30T22:01:34.4784118+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(true) })
2024-09-30T22:01:35.966731+03:00 [WARN] unhandled message: ActiveRepo(SupermavenActiveRepoMessage { repo_simple_name: Some("AutoAnswer") })
2024-09-30T22:01:35.9668013+03:00 [WARN] unhandled message: TaskStatus(SupermavenTaskUpdateMessage { task: "Adapting to 'AutoAnswer'", status: Complete, percent_complete: None })
2024-09-30T22:01:40.5904383+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]
2024-09-30T22:01:43.9610358+03:00 [WARN] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2024-09-30T22:02:17.4891149+03:00 [INFO] open paths []
2024-09-30T22:02:18.8898347+03:00 [INFO] open paths ["\\\\?\\G:\\CodeProjects\\PycharmProjects\\AutoAnswer\\autoanswer\\apps\\jobs\\payment.py"]


<!-- Click above this line and paste or drag-and-drop your log--></pre></details>

--- a/crates/terminal_view/src/terminal_view.rs
+++ b/crates/terminal_view/src/terminal_view.rs
@@ -613,9 +613,8 @@ fn subscribe_for_terminal_events(
     cx: &mut ViewContext<'_, TerminalView>,
 ) -> Vec<Subscription> {
     let terminal_subscription = cx.observe(terminal, |_, _, cx| cx.notify());
-    let terminal_events_subscription = cx.subscribe(terminal, move |this, _, event, cx| {
-        println!("=> {:#?}", event);
-        match event {
+    let terminal_events_subscription =
+        cx.subscribe(terminal, move |this, _, event, cx| match event {
             Event::Wakeup => {
                 cx.notify();
                 cx.emit(Event::Wakeup);
@@ -756,8 +755,7 @@ fn subscribe_for_terminal_events(
                 cx.invalidate_character_coordinates();
                 cx.emit(SearchEvent::ActiveMatchChanged)
             }
-        }
-    });
+        });
     vec![terminal_subscription, terminal_events_subscription]
 }
 
@@ -770,8 +768,24 @@ fn possible_open_paths_metadata(
 ) -> Task<Vec<(PathWithPosition, Metadata)>> {
     cx.background_executor().spawn(async move {
         let mut paths_with_metadata = Vec::with_capacity(potential_paths.len());
-        // \\?\D:\projects\windows-test\main.rs
-        // D:\projects\windows-test\main.rs
+
+        #[cfg(not(target_os = "windows"))]
+        let mut fetch_metadata_tasks = potential_paths
+            .into_iter()
+            .map(|potential_path| async {
+                let metadata = fs.metadata(&potential_path).await.ok().flatten();
+                (
+                    PathWithPosition {
+                        path: potential_path,
+                        row,
+                        column,
+                    },
+                    metadata,
+                )
+            })
+            .collect::<FuturesUnordered<_>>();
+
+        #[cfg(target_os = "windows")]
         let mut fetch_metadata_tasks = potential_paths
             .iter()
             .map(|potential_path| async {

--- a/crates/terminal_view/src/terminal_view.rs
+++ b/crates/terminal_view/src/terminal_view.rs
@@ -613,8 +613,9 @@ fn subscribe_for_terminal_events(
     cx: &mut ViewContext<'_, TerminalView>,
 ) -> Vec<Subscription> {
     let terminal_subscription = cx.observe(terminal, |_, _, cx| cx.notify());
-    let terminal_events_subscription =
-        cx.subscribe(terminal, move |this, _, event, cx| match event {
+    let terminal_events_subscription = cx.subscribe(terminal, move |this, _, event, cx| {
+        println!("=> {:#?}", event);
+        match event {
             Event::Wakeup => {
                 cx.notify();
                 cx.emit(Event::Wakeup);
@@ -755,7 +756,8 @@ fn subscribe_for_terminal_events(
                 cx.invalidate_character_coordinates();
                 cx.emit(SearchEvent::ActiveMatchChanged)
             }
-        });
+        }
+    });
     vec![terminal_subscription, terminal_events_subscription]
 }
 
@@ -768,19 +770,18 @@ fn possible_open_paths_metadata(
 ) -> Task<Vec<(PathWithPosition, Metadata)>> {
     cx.background_executor().spawn(async move {
         let mut paths_with_metadata = Vec::with_capacity(potential_paths.len());
-
+        // \\?\D:\projects\windows-test\main.rs
+        // D:\projects\windows-test\main.rs
         let mut fetch_metadata_tasks = potential_paths
-            .into_iter()
+            .iter()
             .map(|potential_path| async {
-                let metadata = fs.metadata(&potential_path).await.ok().flatten();
-                (
-                    PathWithPosition {
-                        path: potential_path,
-                        row,
-                        column,
-                    },
-                    metadata,
-                )
+                let metadata = fs.metadata(potential_path).await.ok().flatten();
+                let path = PathBuf::from(
+                    potential_path
+                        .to_string_lossy()
+                        .trim_start_matches("\\\\?\\"),
+                );
+                (PathWithPosition { path, row, column }, metadata)
             })
             .collect::<FuturesUnordered<_>>();
 

